packages/connectkit/src/hooks/usePaymentState.ts

```diff
@@ -61,6 +62,8 @@ export interface PayParams {
   preferredChains?: number[];
   /** Preferred tokens. These appear first in the token list. */
   preferredTokens?: { chain: number; address: Address }[];
+  /** Developer metadata. E.g. correlation ID. */
```

Comment:
| User metadata associated with this payment. Can be used eg. for correlated IDs.